src/capi_frontend/capi.cpp

```diff
@@ -273,7 +273,7 @@ DLL_PUBLIC OVMS_Status* OVMS_ServerStartFromConfigurationFile(OVMS_Server* serve
     ovms::Server* srv = reinterpret_cast<ovms::Server*>(server);
     ovms::ServerSettingsImpl* serverSettings = reinterpret_cast<ovms::ServerSettingsImpl*>(server_settings);
     ovms::ModelsSettingsImpl* modelsSettings = reinterpret_cast<ovms::ModelsSettingsImpl*>(models_settings);
-    auto res = srv->start(serverSettings, modelsSettings);
+    auto res = srv->start(serverSettings, modelsSettings, false);
```

Comment:
Currently we have 2 containers for storing configuration:
- server settings
- model settings

You introduce yet another container for holding configuration: bool withPython.

I think it should be moved to server settings

Reply:
Let's do it in a separate PR